src/codegeneration/TemplateLiteralTransformer.js

```diff
@@ -198,6 +199,21 @@ export class TemplateLiteralTransformer extends TempVarTransformer {
     return ParseTreeTransformer.prototype.transformFunctionBody.call(this, tree);
   }
 
+  transformNewExpression(tree) {
+    if (tree.operand) {
+      var operand = tree.operand;
+      do {
+        // If operand contains a tagged TemplateLiteral, parenthesize the entire
+        // operand.
+        if (operand.operand && operand.type === TEMPLATE_LITERAL_EXPRESSION) {
```

Comment:
needs a type check, accessing non existing properties is something we try to not do.